parcels/field.py

```diff
@@ -91,10 +92,24 @@ def __getitem__(self, key):
         return self.eval(*key)
 
     @cachedmethod(operator.attrgetter('interpolator_cache'))
-    def interpolator(self, t_idx):
+    def interpolator2D(self, t_idx):
         return RectBivariateSpline(self.lat, self.lon,
                                    self.data[t_idx, :])
 
+    def interpolator1D(self, idx, time, y, x):
+        # Return linearly interpolated field value:
+        if x is None and y is None:
+            t0 = self.time[idx-1]
```

Comment:
Unfortunately this breaks for `idx=0`, ie. with static fields like we use in the peninsula test.